aqua/qt6/Portfile

```diff
@@ -49,6 +49,12 @@ foreach {qt_base qt_info} ${qt6::available_versions} {
     }
 }
 
+# qt-base is broken on macOS 15+, but that locks us in to this on all of them for now,
```

Comment:
same comment as for the other PR: please correct all occurrences of the portname and rephrase the second part of the comment so that it is meaningful.